rcldotnet/rcldotnet_ret.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016-2018 Esteve Fernandez <esteve@apache.org>
```

Comment:
This duplicates all the error codes that are already in https://github.com/ros2/rcl/blob/master/rcl/include/rcl/types.h

Reply:
Ok. Removed. I used that and also rmw constants